x-pack/plugins/synthetics/public/apps/synthetics/components/common/screenshot/empty_thumbnail.tsx

```diff
@@ -63,14 +63,23 @@ export const EmptyThumbnail = ({
         ...(borderRadius ? { borderRadius } : {}),
       }}
     >
-      {isLoading ? (
-        <EuiLoadingContent
+      {isLoading && animateLoading ? (
+        <EuiSkeletonRectangle
+          data-test-subj="stepScreenshotPlaceholderLoading"
+          isLoading={isLoading}
+          height={height}
+          width={width}
+        >
+          {/* `children` is required by this component, even though we'll replace it. */}
+          <span />
+        </EuiSkeletonRectangle>
```

Comment:
you can't do this for now. since this needs to backported. in 8.7 this doesn't exists.

Reply:
Not intended for 8.7 anymore 😃